Week3/homework/step2-3.js

```diff
@@ -30,14 +30,17 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  let i = 0;
+  if (num > 0) {
+    do {
+      result += str;
+      i++;
+    } while (i < num);
+  }
   return result;
 }
 
-console.log('do-while', repeatStringNumTimesWithDoWhile('abc', 3));
+console.log('do-while', repeatStringNumTimesWithDoWhile('abc', 0));
```

Comment:
Think the problem wanted you to write a guard in the case num < 0. No worries though, looks good 👍